src/mission_util.cpp

```diff
@@ -411,7 +411,7 @@ mission_target_params mission_util::parse_mission_om_target( const JsonObject &j
     if( jo.has_member( "random" ) ) {
         p.random = jo.get_bool( "random" );
     }
-    p.search_range  = get_dbl_or_var( jo, "search_range", false, OMAPX );
+    p.search_range  = dbl_or_var( 2500.0 );
```

Comment:
This is crap, but was the most straightforward way to plug in our new value without needlessly churning lots of C++.

Reply:
`search_range` is read in three places in the entire codebase, surely it should not be *too bad* replacing those three, with a [named] constant?

But sure, if we're trying to change as little c++ as possible, please at least add a comment that this is a misfeature and not some kind of big-brained plan.
```suggestion
    // Legacy code. Ideally this should not be a member field at all, since we hardcode it anyway.
    p.search_range  = dbl_or_var( 2500.0 );
```

Reply:
No longer relevant